src/patterns/webform/examples/errors/index.njk

```diff
@@ -0,0 +1,238 @@
+---
+layout: none
+---
+{% extends "styles/page-template/_template.njk" %}
+{% from "components/question/_macro.njk" import onsQuestion %}
+{% from "components/radios/_macro.njk" import onsRadios %}
+{% from "components/textarea/_macro.njk" import onsTextarea %}
+{% from "components/input/_macro.njk" import onsInput %}
+{% from "components/button/_macro.njk" import onsButton %}
+{% from "components/panel/_macro.njk" import onsPanel %}
+{% from "components/lists/_macro.njk" import onsList %}
+
+{% set pageConfig = {
+    "title": 'Error: Web form - Census 2021',
+    "theme": 'census',
+    "absoluteUrl": pageInfo.url,
+    "headMeta": {
+        "description": 'Census 2021 is a UK wide population survey'
+    },
+    "header": {
+        "logoHref": '#0',
+        "titleLogo": 'census-logo-en',
+        "titleLogoHref": '#0',
+        "titleLogoAlt": 'Census 2021'
+    },
+    "breadcrumbs": {
+        "ariaLabel": 'Back',
+        "itemsList": [
+            {
+                "url": '#0',
+                "id": 'back',
+                "text": 'Back',
+                "attributes": {
+                    "data-attribute": 'Example attribute'
+                }
+            }
+        ]
+    },
+    "footer": {}
+} %}
+{% block main %}
+
+    {% call
+        onsPanel({
+            title: 'There are 5 problems with this page',
+            type: 'error',
+            "dsExample": isPatternLib
+        })
+    %}
+
+    {{
+        onsList({
+            "element": 'ol',
+            "itemsList": [
+                {
+                    "text": 'Select your country',
+                    "index": true,
+                    "url": '#webform-country-error',
+                    "classes": 'js-inpagelink'
+                },
+                {
+                    "text": 'What do you need help with?',
+                    "index": true,
+                    "url": '#webform-help-error',
+                    "classes": 'js-inpagelink'
+                },
+                {
+                    "text": 'Please provide more detail?',
+                    "index": true,
+                    "url": '#webform-detail-error',
+                    "classes": 'js-inpagelink'
+                },
+                {
+                    "text": 'Enter your name',
+                    "index": true,
+                    "url": '#webform-name-error',
+                    "classes": 'js-inpagelink'
+                },
+                {
+                    "text": 'Enter your email address',
+                    "index": true,
+                    "url": '#webform-email-error',
+                    "classes": 'js-inpagelink'
+                }
+            ]
+        })
+    }}
+    
+    {% endcall %}
+    
+    <h1 class="u-fs-xxl u-mt-l">Web form</h1>
+
+    {{ onsRadios({
+        "legend": 'Your country',
```

Comment:
Need content design to confirm, but I'd suggest something like...
```suggestion
        "legend": 'What country do you live in?',
```
...is a better legend. To be consistent with the next fieldset.